### PR TITLE
Login & Register pages

### PR DESCRIPTION
**Pull request by rutu14**
Login Page preview: https://deploy-preview-3--zealous-hawking-9549eb.netlify.app/login-page

Register Page review: https://deploy-preview-3--zealous-hawking-9549eb.netlify.app/register-page

### CODE REVIEW

**Review by archit7448 — APPROVED** [review]
looks good to merge

File: css/login.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.login-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding-inline-start: 2.4rem;
+    padding-inline-end: 2.4rem;
+    padding-top: 4.8rem;
+    padding-bottom: 4.8rem;
```
Comment (sudipkundu999) [inline css/login.css]: use shorthand

File: css/login.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    padding-bottom: 4.8rem;    
+    margin: auto;
+    height: inherit;
+    max-width: 51.2rem;
+}
+
+.login-box {    
+    width: 80%;
+    border-radius: 0.8rem;
+    box-shadow: 0 10px 15px -3px rgba(0,0,0,0.1), 0 4px 6px -2px rgba(0,0,0,0.05);
```
Comment (srishtigautam21) [inline css/login.css:16]: just a suggestion you can use px here

**Review by sudipkundu999 — APPROVED** [review]
UI looks clean! Good to merge!

**Review by srishtigautam21 — APPROVED** [review]
good to merge....and nice name of your site

File: css/login.css (hunk 1 above)
Comment (rutu14) [reply css/login.css]: Okay

File: css/login.css (hunk 2 above)
Comment (rutu14) [reply css/login.css:16]: Sure.

**Pull request merged**